types/externs.d.ts

```diff
@@ -148,10 +151,15 @@ declare global {
       gatherMode?: boolean | string;
       /** Flag indicating that the browser storage should not be reset for the audit. */
       disableStorageReset?: boolean;
-      /** How emulation (useragent, device screen metrics, touch) should be applied. `none` indicates Lighthouse should leave the host browser as-is. */
-      emulatedFormFactor?: 'mobile'|'desktop'|'none';
-      /** Dangerous setting only to be used by Lighthouse team. Disables the device metrics and touch emulation that emulatedFormFactor defines. Details in emulation.js */
-      internalDisableDeviceScreenEmulation?: boolean
+
+      // TODO: formFactor should really be required, but since this type is shared in Flags, we can't reasonably do that. We'll likely need to separate SharedFlagsSettings into CLI Flags and Config Settings.
+      /** How Lighthouse should interpret this run in regards to scoring performance metrics and skipping mobile-only tests in desktop. Must be set even if throttling/emulation is being applied outside of Lighthouse. */
+      formFactor?: 'mobile'|'desktop';
+      /** Screen emulation properties (width, height, dpr, mobile viewport) to apply or an object of `{disabled: true}` if Lighthouse should avoid applying screen emulation. It's typically set to disabled if either emulation is applied outside of Lighthouse, or it's being run on a mobile device. For desktop, we recommend applying consistent desktop screen emulation. */
+      screenEmulation?: ScreenEmulationSettings;
```

Comment:
this would become 
```suggestion
      screenEmulation?: Partial<ScreenEmulationSettings>;
```
with the above suggestion.